contents/handbook/company/team/paolo-damico.md

```diff
@@ -4,8 +4,18 @@ sidebar: Handbook
 showTitle: true
 ---
 
+## Q1 2022 Goals
+- ⭐️ Build and share the **vision** for one ambitious and very **exciting** goal.
+- ⭐️ Build the strategy for one **ambiguous opportunity** to improve a company metric or achieve a company goal.
```

Comment:
Not quite sure how to add this in, but this should be something that's ready for people to start working on by the end of Q1.

Reply:
```suggestion
- ⭐️ Build the strategy for one **ambiguous opportunity** to improve a company metric or achieve a company goal (_should be ready to be worked on by end of Q1_)
```